src/main/java/com/project/house/rental/entity/District.java

```diff
@@ -1,4 +1,37 @@
 package com.project.house.rental.entity;
 
+
+import jakarta.persistence.*;
+import lombok.*;
+import lombok.experimental.FieldDefaults;
+
+import java.util.Date;
+
+@Getter
+@Setter
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+@Entity
+@Table(name = "districts")
+@FieldDefaults(level = AccessLevel.PRIVATE)
```

Comment:
Thêm Annotion để soft delete

Ví dụ: 
@SQLDelete(sql = "UPDATE table_product SET deleted = true WHERE id=?")
@Where(clause = "deleted=false")

Trong đó SQLDelete sẽ chạy khi JPA delete. Thay thế hàm delete mặc định. Ở đây sẽ update cái isDelete.
Cái Where sẽ apply cho toàn bộ query, để tìm ra những record có isDelete = false